physionet-django/console/views.py

```diff
@@ -1109,7 +1118,9 @@ def user_management(request, username):
                                                             'emails': emails,
                                                             'projects': projects,
                                                             'training_list': training,
-                                                            'credentialing_app': credentialing_app})
+                                                            'credentialing_app': credentialing_app,
+                                                            'aws_info': aws_info,
```

Comment:
Instead of getting separate variables for each cloud ID (`aws_info`, `gcp_info`), we could just pass the `CloudInformation` information object and then access it in the template with `cloud_object.aws_id`, `cloud_object.gcp_email` etc, but I think your approach is fine for now.